searcher/searcher.go

```diff
@@ -211,6 +211,12 @@ func MakeAll(cfg *config.Config) (map[string]*Searcher, map[string]error, error)
 	return searchers, errs, nil
 }
 
+func PollAll(cfg *config.Config, searchers map[string]*Searcher) {
```

Comment:
Sounds better, agreed. Made the suggested change. How do you feel about me adding vcsDir and name to the searcher struct?